content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md

```diff
@@ -387,6 +387,110 @@ Allows fine-grained controls of various features only available in the enterpris
 
       - The size of the in-memory Artemis buffer for messages, in bytes. Note that there is a lower bound to the buffer size, which is calculated based on the maximum message size of the network parameters to ensure messages of any allowed size can be stored successfully. As a result, any value lower than this bound will be ignored with the appropriate logging. This bound is also used as the default, if no value is specified.
 
+* `ledgerRecoveryConfiguration`
+
+    * This configuration allows you to tailor ledger recovery behavior for your Corda node. It offers flexibility in defining
+    parameters related to key pair pre-generation, backup intervals, and confidential identity backup options.
```

Comment:
Just thinking, as this is a fairly big topic. What about adding something like the following to the end of this description. Where you link to the doc that Jose and Balwant wrote.
Please see XXXX for a detailed description of ledger recovery that makes use of this configuration.

Reply:
Ok, I'll add the reference later and leave this PR open until I merge the ledger recovery one as I still haven't decided on the location of this content.

Reply:
Added link